src/layers/legacy/medVtkInria/vtkImageView/vtkImageView2D.cxx

```diff
@@ -456,8 +458,14 @@ void vtkImageView2D::UpdateDisplayExtent()
   int *range = this->GetSliceRange();
   if (range)
   {
-    slice = std::max (slice, range[0]);
-    slice = std::min (slice, range[1]);
+      if (std::isdigit(range[0]))
```

Comment:
What is the goal of this test? The `isdigit` function tests for `1` up to `9` only.

Reply:
(I meant `0` to `9`)